lib/codegen/fromcto/java/javavisitor.js

```diff
@@ -278,7 +278,12 @@ class JavaVisitor {
                 parameters.fileWriter.writeLine(1, '}');
                 break;
             default:
-                parameters.fileWriter.writeLine(1, 'private ' + fieldType + ' ' + fieldName + ';' );
+                if (this.isMap(field)) {
+                    const decl = field.getModelFile().getAllDeclarations().find(d => d.name === field.ast.type.name);
```

Comment:
Assumes type of map value in same model as map decl.

Reply:
Now gets the declaration directly from ModelFile